src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -204,3 +204,74 @@ Given an event, a predefined action will be applied on the workflow execution.
 If an inclusion or exclusion calendar is not retrievable, it is blocking the <<_glossary_workflow,*Workflow*>> submission.
 An inclusion or exclusion calendar can become not retrievable if it cannot be downloaded from its URL and the
 <<_glossary_job_planner,*Job Planner*>> cache doesn't hold a copy.
+
+== Notifications
+
+The <<_glossary_job_planner,*Job Planner*>> can send notifications to users when particular events occur. The <<_glossary_notification_service,*Notification service*>> documentation can be found link:../admin/ProActiveAdminGuide.html#_notification_service[here].
+
+To start receiving notifications from the <<_glossary_job_planner,*Job Planner*>>, you must:
+
+- Configure your Job planner's <<_glossary_notification_subscription,*Subscription*>> as it is described in link:../user/ProActiveUserGuide.html#_service_subscriptions[Service subscriptions documentation] of the <<_glossary_notification_service,*Notification service*>>.
+- Inside the Job planner subscription, select from available events the ones you wish to be notified of.
+Available Job planner events with their description and example values are detailed in the table in link:../user/ProActiveUserGuide.html#_job_planner_notifications[Job Planner event types].
+
+== Job Planner Configuration
+
+<<_glossary_job_planner,*Job Planner*>> provides features to specify actions it will take regarding different situations that it can encounter when submitting <<_glossary_workflow,*Workflows*>> according to a <<_glossary_calendars_definition,*Calendar*>>.
+
+This configuration is done in the properties file located `PROACTIVE_HOME/dist/war/job-planner/WEB-INF/classes/application.properties`
+
+=== Planned execution(s) has been missed
+This configuration controls how the <<_glossary_job_planner,*Job Planner*>> reacts when it detects that one or more executions
+have been missed for a given association. For instance if the server is down for a period of time, the planned execution will not take place.
+Two properties are used to define how <<_glossary_job_planner,*Job Planner*>> reacts to this situation.
+----
+pa.on_skipped_execution.submit=true
+----
+True by default, this option tells the <<_glossary_job_planner,*Job Planner*>> to submit a new execution if it detects that previous executions have been missed.
+It is equivalent to say that the submission has been delayed and thus a *Delayed submission executing* notification will be triggered for all users subscribed to this event.
+If set to false, <<_glossary_job_planner,*Job Planner*>> will not submit a new execution.
+
+The second property defines the delay in milliseconds from a missed execution and the current time for which the <<_glossary_job_planner,*Job Planner*>> will submit a delayed execution.
```

Comment:
This property, when greater than zero, allows the job planner to submit a missed execution within a given time window in milliseconds.  If the job planner is unable to submit the missed execution after the configured delay,  it will be skipped until the next calendar occurrence. By default, the skipped execution window is disabled : `-1`.